exploring_UV_extinction_curves_pt2/Exploring_UV_extinction_curves_pt2.ipynb

```diff
@@ -0,0 +1,746 @@
+{
```

Comment:
<p>This is so cool!! I feel like it's worth some text commenting here. Because now I can really see the science! You could draw the reader's attention to how the reddening is much pronounced at higher 1/lambda (lower wavelength).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>